indico/web/templates/forms/multiple_tag_select_widget.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <div id="{{ field.id }}-wrapper"></div>
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        setupMultipleTagSelectWidget({
+            fieldId: {{ field.id | tojson }},
+            wrapperId: "{{ field.id }}-wrapper",
+            choices: {{ field.choices | tojson }},
+        });
+    </script>
+{% endblock %}
```

Comment:
missing trailing line break (make sure to configure your editor to add those automatically)